portability-transfer/src/main/java/org/datatransferproject/transfer/copier/PortabilityAbstractInMemoryDataCopier.java

```diff
@@ -205,6 +227,34 @@ private void importIteration(
     }
   }
 
+  private void sizeCalculationIteration(UUID jobId, String jobIdPrefix,
+      DataModel exportedData) throws CopyException {
+    Collection<? extends DownloadableItem> items;
```

Comment:
(shouldn't block PR) another future consideration is make `getItems` an interface all models are required to have so we don't have to maintain this coverage. We should definitely at least come back to this code after you merge and drop a TODO here to fill out the coverage (either via interface or by hand for the current known models).